introduction/deprecated_features/README.md

```diff
@@ -0,0 +1,22 @@
+# Deprecated Features
+
+This page lists deprecated features in CSM. They are no longer being actively developed and are planned to be removed in a future CSM release.
+This groups the deprecated features by the CSM release in which they were deprecated, in reverse chronological order (the most recently deprecated
+features are listed first).
+
+## Deprecated in CSM 1.2
+
+* HMNFD v1 REST API has been deprecated as of CSM version 1.2. The v1 HMNFD APIs are targeted for removal in the CSM 1.5 release.
```

Comment:
@nieuwsma should `CSM 1.5` be aligned with to one of our non-optional releases like CSM 1.6?

Reply:
Definitely it seems like it makes more sense to target the removal for a non-optional release.

Reply:
Are there other consumers of this API outside of CSM?

Reply:
potentially.

Reply:
Let me know what the consensus is here and I will edit accordingly.

Reply:
Ok -- let me know what version I should specify here.

Reply:
since the only recipe that is coming after 1.3 is 1.6 (Alice) I would assume that is the one

Reply:
Ok, I'll make that change. Just need re-review from Jason S now for this to be ready to merge.